### PR TITLE
Align previous link style with Breadcrumbs

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Change previous links to use a single breadcrumb instead. Needs [this PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/342) to be merged to update to the latest version of the design system first

### How to review 
Check previous links have been updated with the correct style and they work exactly as they did previously.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Issue #802: Align previous link used in questionnaires with breadcrumbs, and consider use of back links in transactions](https://trello.com/c/UOFhPI2I/4186-issue-802-align-previous-link-used-in-questionnaires-with-breadcrumbs-and-consider-use-of-back-links-in-transactions)

File: templates/layouts/_base.html [hunk 1]
```diff
@@ -8,6 +8,24 @@
...
+      {
+        "url": previous_location_url,
+        "id": "top-previous",
+        "text": _("Previous"),
+          "attributes": {
+            "data-ga": 'click',
+            "data-ga-category": 'Navigation',
+            "data-ga-action": 'Previous link click'
+          }
+      }
```
Comment (ajmaddaford) [inline templates/layouts/_base.html:24]:
```suggestion
      {
        "url": previous_location_url,
        "id": "top-previous",
        "text": _("Previous"),
        "attributes": {
          "data-ga": 'click',
          "data-ga-category": 'Navigation',
          "data-ga-action": 'Previous link click'
        }
      }
```

**Review by ajmaddaford — APPROVED** [review]

**Review by LJBabbage — CHANGES_REQUESTED** [review]
I wasn't expected such a small font, its coming up as 9.33pt, is that right?

image

**Comment by rmccar** [comment]
> I wasn't expecting such a small font, it's coming up as 9.33pt, is that right?
> 
> image
> 

Yeah this is the style that is used for breadcrumbs and back links will now look the same

**Review by LJBabbage — APPROVED** [review]
Cool, all good then

**Pull request merged**